homeassistant/components/alarmdecoder.py

```diff
@@ -71,7 +74,9 @@
     vol.Required(CONF_ZONE_NAME): cv.string,
     vol.Optional(CONF_ZONE_TYPE,
                  default=DEFAULT_ZONE_TYPE): vol.Any(DEVICE_CLASSES_SCHEMA),
-    vol.Optional(CONF_ZONE_RFID): cv.string})
+    vol.Optional(CONF_ZONE_RFID): cv.string,
+    vol.Optional(CONF_RELAY_ADDR): cv.byte,
+    vol.Optional(CONF_RELAY_CHAN): cv.byte})
```

Comment:
Is this required for the relay address to be effective? If so, mark both items with `vol.Inclusive`.